waspc/examples/todoApp/src/dbSeeds.ts

```diff
@@ -34,14 +35,7 @@ async function createUser(prismaClient: PrismaClient, data: any) {
     },
   })
 
-  if (newUser.auth?.identities) {
-    newUser.auth.identities = newUser.auth.identities.map((identity) => {
-      identity.providerData = JSON.parse(identity.providerData)
-      return identity
-    })
-  }
-
-  return newUser
+  return createAuthUser(newUser as any)
```

Comment:
I wanted to get this code to work, I wanted to make it work without an assertion but it proved difficult. I'll give it an another try.

Reply:
I've now hidden this API since it would expose some server imports on the client. Now this test is written a bit differently with the `{} as AuthUser` method which works well for these kinds of tests.